runner/ansible/roles/1.3.5/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.3.5'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    DEF_WDTIMEOUT={{ expected[test_id] }}
```

Comment:
I have some doubts on this script.
As far as I see, it iterates over to store a result in `result_watchdog`, but every iteration this value is overwritten.
So if for some reason the 1st sdb device has the wrong value, but the 2nd the correct one, this check will still pass, right?
**Most probably you just inherited this script, I understand this, but I'm wondering if we can make it more robust for this circunstance**

Reply:
Yes, this is how the current test is run. And yes, it is not a good test, but no. After initial setting
result_wdtimeout is only updated if:
if [[ "${wdtimeout}" -ne "${DEF_WDTIMEOUT}" ]]; then

What is the case is if there are no entries in sbdarray then the code still returns the expected value. 

I think for now we should track what is in examples/generic-azure.yaml and track what is in. I can work on fixing that test, but I think that should be separate from this review.

Reply:
If this is the case, let's remove this test at all by now.
I know that you just translated the currently existing tests, but I would prefer to only bring the ones that give us confidence about what they do

Reply:
I have synced what is in 1.3.5 ansible with what is in example.
There was a coding error that caused the test to always pass in ansible. This may have been in the code
we started with, or we may have introduced the error when we did the ansible.That has been fixed. So, the test does what it is intended to do, but is not as bullet proof as it could be.
if sbdarray has no values, then I don't think that we will execute the loop and if that is the case result_wdtimeout will be left with the initial assignment of result_wdtimeout=${DEF_WDTIMEOUT} which would log a pass. I am not sure that can actually happen unless stonith is misconfigured, i.e.  /etc/sysconfig/sbd is corrupt or missing. I have not seen that is testing, but could be possible. But the current test is working.

Reply:
Ok, let's move forward with this. If it needs changes we will get them in the next iterations